app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :name,  presence: true,
+                    length: { maximum: Settings.user.length_of_name }
```

Comment:
bên trong {} ko để space nak
```suggestion
                    length: {maximum: Settings.user.length_of_name}
```

Reply:
sửa những chỗ còn lại luôn